lms/templates/fields/field_textarea.underscore

```diff
@@ -1,6 +1,10 @@
 <div class="wrapper-u-field">
     <div class="u-field-header">
-        <label class="u-field-title" for="u-field-textarea-<%- id %>" id="u-field-title-<%- id %>" aria-describedby="u-field-message-help-<%- id %>"></label>
+        <% if (mode === 'edit') { %>
+            <label class="u-field-title" for="u-field-textarea-<%- id %>" id="u-field-title-<%- id %>" aria-describedby="u-field-message-help-<%- id %>"></label>
+        <% } else { %>
+            <span class="u-field-title" id="u-field-title-<%- id %>" aria-describedby="u-field-message-help-<%- id %>"></span>
```

Comment:
@cptvitamin 

1. I won't add the aria-describedby attributes if there is no message.
2. Making it a <h3> wouldn't work well because this field can be used in multiple locations, and h3 may not always be appropriate. For example, this component is also used in the form to create a new team (for the description field).
3. Yes, they do.